src/libs/actions/Policy.ts

```diff
@@ -3565,6 +3565,37 @@ function enablePolicyTags(policyID: string, enabled: boolean) {
 }
 
 function enablePolicyTaxes(policyID: string, enabled: boolean) {
+    const taxRatesData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    taxRates: DEFAULT_TAXES_OPTIMISTIC_DATA,
```

Comment:
I think we can assign `DEFAULT_TAX_RATES` to a variable and explicitly set the `pendingAction` for `id_TAX_EXEMPT` and `id_TAX_RATE_1` before setting to `taxRates` here.